porcupine/plugins/run/dialog.py

```diff
@@ -105,8 +105,16 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
             validated_callback=self.update_run_button,
         )
 
+        ttk.Label(content_frame, text="Substitutions:").pack(anchor="w")
+
         sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_textbox = textutils.create_passive_text_widget(
+            content_frame, height=len(substitutions), width=1
+        )
+        sub_textbox.pack(fill="x", padx=15, pady=(0, 20))
```

Comment:
For long file paths, it's better to have the padding only on the left side.
```suggestion
        sub_textbox.pack(fill="x", padx=(15, 0), pady=(0, 20))
```